src/site/content/en/lighthouse-pwa/without-javascript/index.md

```diff
@@ -1,55 +1,55 @@
-largest possible audience. The most common definition of progressive
-enhancement is as follows:
-
-Basic content and page functionality should rely on
-only the most fundamental web technologies, to ensure that the page is usable
-across all browsing conditions. Enhanced experiences, such as sophisticated
-styling using CSS, or interactivity using JavaScript, can be layered on top for
-the browsers that support those technologies. But basic content and page
+largest possible audience. The core principle is that
+basic content and page functionality should rely on
+only the most fundamental web technologies.
+Enhanced experiences, such as sophisticated
+styling using CSS or interactivity using JavaScript, can be layered on top for
+browsers that support those technologies. But basic content and page
 functionality should not rely on CSS or JavaScript.
 
-Lighthouse flags pages that don't contain some content
+## How this Lighthouse audit fails
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/)
+flags pages that don't contain _some_ content
 when JavaScript is unavailable:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="without-javascript.png" alt="Lighthouse audit showing page doesn't contain some content when JS is unavailable">
-  <figcaption class="w-figcaption">
-    Page doesn't contain some content when JS is unavailable.
-  </figcaption>
+  <img class="w-screenshot" src="without-javascript.png" alt="Lighthouse audit showing page doesn't contain some content when JS is unavailable">
 </figure>
 
-## How this audit fails
-
 Lighthouse disables JavaScript on the page and then inspects the page's HTML. If
-the HTML is empty then the audit fails. If the HTML is not empty then the audit
-passes.
+the HTML is empty, the audit fails.
 
 {% include 'content/lighthouse-pwa/scoring.njk' %}
 
-## Recommendations
+## How to ensure your page has content without JavaScript
 
 Progressive enhancement is a large and contentious topic. One camp says that,
 in order to adhere to the strategy of progressive enhancement, pages should
-be layered so that basic content and page functionality only require HTML. See
+be layered so that basic content and page functionality only require HTML.
+See Smashing Magazine's
 [Progressive Enhancement: What It Is, And How To Use It](https://www.smashingmagazine.com/2009/04/progressive-enhancement-what-it-is-and-how-to-use-it/)
 for an example of this approach.
 
 Another camp believes that this strict approach is unfeasible or unnecessary
 for many modern, large-scale web applications and suggests using inline
-critical path CSS in the document `<head>` for absolutely critical page styles.
-See [Critical Rendering Path](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/) for more on this approach.
+critical-path CSS in the document `<head>` for absolutely critical page styles.
+See Google's
```

Comment:
I wish I'd caught this phrasing in the original audit doc. Because the only link mentioned is Google's critical render path docs, it sort of implies that Google is in the camp saying progressive enhancement is unfeasible, which I don't think is true. The critical rendering path doc is mostly explaining how things load and doesn't mention progressive enhancement.

Since it's 5:16 pm, my brain doesn't have a good solution to this. But I wanted to flag it so we can discuss.

Reply:
Apologies, this was one of the last posts I edited, so I didn't actually read that doc. I think this is an easy fix though. We have a link to the doc in the Resources section, so I'm just going to omit this sentence. A side benefit to doing that is it doesn't give further validation (via citation) to a position the site finds problematic.